pxr/usd/usd/wrapValidator.cpp

```diff
@@ -57,4 +92,28 @@ void wrapUsdValidator()
         .add_property("schemaTypes", make_getter(
             &UsdValidatorMetadata::schemaTypes, return_value_policy<TfPySequenceToList>()))
         .def_readonly("isSuite", &UsdValidatorMetadata::isSuite);
+
+    TfPyRegisterStlSequencesFromPython<UsdValidationError>();
+    UsdValidationErrorVector (UsdValidator::*_ValidateLayer)(
+        const SdfLayerHandle &layer) const = &UsdValidator::Validate;
+    UsdValidationErrorVector (UsdValidator::*_ValidateStage)(
+        const UsdStagePtr &stage) const = &UsdValidator::Validate;
+    UsdValidationErrorVector (UsdValidator::*_ValidatePrim)(const UsdPrim &prim)
+        const = &UsdValidator::Validate;
+    class_<UsdValidator, boost::noncopyable>("Validator", no_init)
```

Comment:
I think that makes sense to me, but to confirm, @nvmkuruc your suggestion will help a developer do:

```
validator = UsdValidatorRegistry.GetOrLoadValidatorByName("plug:validatorName")
print(validator)
UsdValidator(name='plug:validatorName')
```

Instead of printing the python object?

Reply:
`__repr__` added. Do we need to add method to stringify validator?

Reply:
@tallytalwar Yup! It's analogous to `Sdf.Layer.FindOrOpen`.

I don't think stringify has much of a use case.